File: typescript/core/solution/isStringUpper.ts
export function isUpperCase(str: string) { 
  let counter:number = 0;
  let result:boolean = false;
  str.split('').forEach(value => 
  { 
    if(value === value.toUpperCase()){  
    }else{
      counter = counter + 1; 
    }
  }); 

  if(counter === 0){
    result = true;
  }else{
    result = false;
  }
  return result;
}

File: python/core/solution/isStringUpper.py
def is_uppercase(inp):  
    counter = 0
    all_uppercase = False
    for value in inp:
        if value.isalpha() and not value.isupper(): 
            counter = counter + 1  

    if counter == 0:
        all_uppercase = True;
        print("True")
    else:
        all_uppercase = False;
        print("False")
    return all_uppercase;
